<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>NewTableWindow</class>
 <widget class="QDialog" name="NewTableWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <widget class="QLabel" name="label_4">
     <property name="text">
      <string>Create New Table</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Table name:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="tableNameLineEdit"/>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>Column Name: </string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="columnNameLineEdit"/>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Select column data type</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="intRadioButton">
       <property name="text">
        <string>Integer</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="FloatRadioButton">
       <property name="text">
        <string>Float</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="charRadioButton">
       <property name="text">
        <string>Char String</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="stringSizeLayout">
     <item>
      <widget class="QLabel" name="stringSizeLabel">
       <property name="text">
        <string>String Size:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="stringSizeLineEdit"/>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_4">
     <item>
      <widget class="QCheckBox" name="isPKCheckBox">
       <property name="text">
        <string>Is Primary Key</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="addColumnButton">
       <property name="text">
        <string>Add Column</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="doneButton">
       <property name="text">
        <string>Done</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="cancelButton">
       <property name="text">
        <string>Cancel</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
